contracts/SlashIndicator.sol

```diff
@@ -46,13 +44,13 @@ contract SlashIndicator is ISlashIndicator, System, IParamSubscriber, IApplicati
     uint256 public felonySlashScope;
 
     event validatorSlashed(address indexed validator);
-    event maliciousVoteSlashed(bytes32 indexed voteAddrSlice);
+    event maliciousVoteSlashed(bytes32 indexed voteAddrSlice);  // @dev deprecated
```

Comment:
I will suggest we put the `deprecated` event together, like
```
event maliciousVoteSlashed(bytes32 indexed voteAddrSlice);  // @dev deprecated
event knownResponse(uint32 code);  // @dev deprecated
event unKnownResponse(uint32 code);  // @dev deprecated
event crashResponse();  // @dev deprecated
```

Reply:
moved for all events